src/components/UserRegistration/UserRegistration.tsx

```diff
@@ -32,19 +34,22 @@ class UserRegistration extends Component<UserRegistrationProps & UserRegistratio
     }
 
     public render() {
-        return <form className="UserRegistration" onSubmit={(e) => {e.preventDefault(); this.onSubmit(e); }}>
-        <p>
-           <label>Username: <input type="text" name="username" required/></label>
-        </p>
-        <p>
-            <label>Email: <input type="email" name="email" required/></label>
-        </p>
-        <p>
-            <label>Password: <input type="password" name="password" required/></label>
-        </p>
-        <p>
-            <button type="submit">Register</button>
-        </p>
+        return <form className="UserRegistration" onSubmit={(e) => {
+            e.preventDefault();
+            this.onSubmit(e);
+        }}>
+            <p>
+                <label>Username: <input type="text" name="username" required/></label>
+            </p>
+            <p>
+                <label>Email: <input type="email" name="email" required/></label>
```

Comment:
Same here. Maybe some more frontend checking than just required? Or does the backend do that very well?